base/linalg/blas.jl

```diff
@@ -61,16 +61,12 @@ import ..LinAlg: BlasReal, BlasComplex, BlasFloat, BlasInt, DimensionMismatch, c
 
 # utility routines
 function vendor()
-    try
-        cglobal((:openblas_set_num_threads, Base.libblas_name), Void)
+    lib = Libdl.dlopen(Base.libblas_name)
```

Comment:
Make this `Libdl.dlopen_e` too? This function wasn't throwing an error if the lib is not found before?